Week1/assignments/armand-collins/Palindrome.js

```diff
@@ -0,0 +1,25 @@
+
+const palindrome = (strn) => {
+  let str = strn.toLowerCase();
+  let first, midChar, end;
+  let mid = Math.round(str.length/2)
+
+  if(str.length % 2 != 0) {
+    midChar = str.slice(mid-1, mid)
+  }
+
+  first = str.slice(0, mid-1);
+  end = str.slice(mid);
+  newFirst = first.split('').reverse().join('');
+
+  if( newFirst == end) {
```

Comment:
Lines 15 to lines 19 could be made into a one liner as such: `return newFirst === end`
Good solution :-)